ios/ReactNativeCameraKit/RealCamera.swift

```diff
@@ -149,21 +159,20 @@ class RealCamera: NSObject, CameraProtocol, AVCaptureMetadataOutputObjectsDelega
         }
     }
 
-    func update(zoomScale: CGFloat) {
-        guard !zoomScale.isNaN else { return }
+    func update(pinchVelocity: CGFloat, pinchScale: CGFloat) {
```

Comment:
I see. I had re-written it to use scale without velocity (so it's always linear to pinch size, and you can zoom far in and out at the cost of accuracy). That's why you see it. I then realized that the native camera uses velocity and tried to match it, however velocity is very finicky and as you zoom further in, it becomes harder to zoom out, esp if the phone is sluggish. Not clear what math the native camera uses.
I'm fine with going with 20 for now, but it needs to be well tested, otherwise we should use the scale-based zooming.

Reply:
What makes you feel they are using velocity? 
Native zoom seems to be capped at x10 which is roughly a zoomFactor of 20 on my iPhone (while `videoMaxZoomFactor` is like 180). Image at 20 is already kinda bad, not sure why they allow 180

Using scale + capping at 20 feels fairly similar to the native app. Miles better than all attempts I did with velocity. I pushed it so you can try. Let me know

PS: resetting `pinchRecognizer.scale` to `1.0` every time value changes is key to make the algorithm work

Reply:
Yeah maybe. At first it looked like it was using velocity but after further inspection and using the grid on the native camera to align my fingers, it looks like there's no velocity involved.
In that case, we do need the `pinchRecognizer.scale` and the original code I wrote to use scale to zoom.